README.md

```diff
@@ -37,42 +37,81 @@ DeepRank2 extensive documentation can be found [here](https://deeprank2.rtfd.io/
   - [Overview](#overview)
   - [Table of contents](#table-of-contents)
   - [Installation](#installation)
-    - [Dependencies](#dependencies)
-    - [Deeprank2 Package](#deeprank2-package)
-    - [Test installation](#test-installation)
+    - [Dockerfile](#dockerfile)
+    - [Non-pythonic dependencies](#non-pythonic-dependencies)
+    - [Pythonic dependencies](#pythonic-dependencies)
+      - [Test installation](#test-installation)
     - [Contributing](#contributing)
-    - [Data generation](#data-generation)
-    - [Datasets](#datasets)
-      - [GraphDataset](#graphdataset)
-      - [GridDataset](#griddataset)
-    - [Training](#training)
+  - [Data generation](#data-generation)
+  - [Datasets](#datasets)
+    - [GraphDataset](#graphdataset)
+    - [GridDataset](#griddataset)
+  - [Training](#training)
   - [Computational performances](#computational-performances)
   - [Package development](#package-development)
 
 ## Installation
 
-The package officially supports ubuntu-latest OS only, whose functioning is widely tested through the continuous integration workflows. 
+Note that the package officially supports ubuntu-latest OS only, whose functioning is widely tested through the continuous integration workflows. 
 
-### Dependencies
+### Dockerfile
 
-Before installing deeprank2 you need to install some dependencies. We advise to use a [conda environment](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) with Python >= 3.10 installed. The following dependency installation instructions are updated as of 14/09/2023, but in case of issues during installation always refer to the official documentation which is linked below:
+In order to try out the package without worrying about your OS and without the need of installing all the required dependencies, we created a `Dockerfile` that can be used for taking care of everything in a suitable container. After having cloned the repository and installed [Docker](https://docs.docker.com/engine/install/), run the following commands from the root of the repository.
+
+Build the Docker image:
+```bash
+docker build -t deeprank2 .
+```
+
+SSH to a running container:
+
+```bash
+docker run -it --expose 3000  -p 3000:3000 deeprank2
+```
+
+Run the tutorials' notebooks from within the running container:
+```bash
+cd tutorials
+jupyter notebook --ip 0.0.0.0 --no-browser --allow-root --port 3000
+```
+
+Now you can run the tutorials' notebook. More details about their content can be found [here](https://github.com/DeepRank/deeprank2/blob/main/tutorials/TUTORIAL.md). Note that in the docker container only the raw PDB files are downloaded, needed as a starting point for the tutorials. You can obtain the processed HDF5 files by running the `data_generation_xxx.ipynb` notebooks. Because Docker containers are limited in memory resources, we limit the number of data points processed in the tutorials'. Please install the package locally to fully leverage its capabilities.
+
```

Comment:
When a user has finished testing it, should they then remove the docker image? Maybe also add instructions on how to do that, as it is quite a large thing to have stored on HD if not used anymore.

Reply:
I don't really agree with this. I think, especially if we want to make this a widely used tool, the installation instructions should be as dummy proof as possible. We don't need detailed instructions for this or above (and below), but it doesn't hurt to have a few words on this.

Reply:
I added some general info and refs about that